server/src/main/java/org/apache/druid/indexing/overlord/SegmentPublishResult.java

```diff
@@ -100,6 +118,7 @@ public String toString()
     return "SegmentPublishResult{" +
            "segments=" + segments +
            ", success=" + success +
+           ", exception=" + errorMsg +
```

Comment:
Maybe we should keep the original name of the member variable and enclose the string `errorMsg` with single quotes.

```java
@Override
public String toString()
{
  return "SegmentPublishResult{" +
         "segments=" + segments +
         ", success=" + success +
         ", errorMsg='" + errorMsg + '\'' +
         '}';
}
```

Reply:
Thanks, fixed!

Reply:
You are welcome.